packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -663,19 +664,29 @@
 
 @mixin bpk-blockquote {
   margin: 0 0 $bpk-spacing-sm 0;
-  padding: $bpk-spacing-sm;
-  border-left: solid $bpk-spacing-sm $bpk-color-gray-50;
+  padding: $bpk-spacing-xs $bpk-spacing-xs $bpk-spacing-xs $bpk-spacing-md;
+  color: $bpk-color-gray-500;
+  font-style: italic;
+
+  @include bpk-border-left-lg($bpk-color-blue-500);
 
   @include bpk-rtl {
-    border-right: solid $bpk-spacing-sm $bpk-color-gray-50;
+    padding: $bpk-spacing-xs $bpk-spacing-md $bpk-spacing-xs $bpk-spacing-xs;
     border-left: 0;
+
+    @include bpk-border-right-lg($bpk-color-blue-500);
   }
 
   > *:last-child {
     margin-bottom: 0;
   }
 }
 
+@mixin bpk-blockquote-extra-spacing {
```

Comment:
This should be a modifier so `bpk-blockqoute--extra-spacing` as it modifies an existing block element rather than being a distinct element from `bpk-blockquote`